cfme/tests/infrastructure/test_webmks_console.py

```diff
@@ -52,9 +51,21 @@ def configure_vmware_console_for_test(appliance, provider):
     settings_pg.update()
 
 
+@pytest.yield_fixture
+def ssh_client(vm_obj, provider):
+    """Provide vm_ssh_client for ssh operations in the test."""
+    console_vm_username = credentials[provider.data.templates.get('console_template')
```

Comment:
I think this needs to take console_template as a fixture instead of looking it up through provider. End up with the same AttrDict, but this way its more explicit that the same template is needed both here and for the test using ssh_client.